flex-templates/java/regional_dlp_de_identification/src/main/java/org/apache/beam/samples/DLPTextToBigQueryStreaming.java

```diff
@@ -0,0 +1,780 @@
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.coders.KvCoder;
+import org.apache.beam.sdk.coders.StringUtf8Coder;
+import org.apache.beam.sdk.io.Compression;
+import org.apache.beam.sdk.io.FileIO;
+import org.apache.beam.sdk.io.FileIO.ReadableFile;
+import org.apache.beam.sdk.io.ReadableFileCoder;
+import org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO;
+import org.apache.beam.sdk.io.range.OffsetRange;
+import org.apache.beam.sdk.metrics.Distribution;
+import org.apache.beam.sdk.metrics.Metrics;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.Validation.Required;
+import org.apache.beam.sdk.options.ValueProvider;
+import org.apache.beam.sdk.options.ValueProvider.NestedValueProvider;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.GroupByKey;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.transforms.View;
+import org.apache.beam.sdk.transforms.Watch;
+import org.apache.beam.sdk.transforms.WithKeys;
+import org.apache.beam.sdk.transforms.splittabledofn.OffsetRangeTracker;
+import org.apache.beam.sdk.transforms.splittabledofn.RestrictionTracker;
+import org.apache.beam.sdk.transforms.windowing.AfterProcessingTime;
+import org.apache.beam.sdk.transforms.windowing.FixedWindows;
+import org.apache.beam.sdk.transforms.windowing.Repeatedly;
+import org.apache.beam.sdk.transforms.windowing.Window;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PCollectionView;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+import org.joda.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * The {@link DLPTextToBigQueryStreaming} is a streaming pipeline that reads CSV
+ * files from a
+ * storage location (e.g. Google Cloud Storage), uses Cloud DLP API to inspect,
+ * classify, and mask
+ * sensitive information (e.g. PII Data like passport or SIN number) and at the
+ * end stores
+ * obfuscated data in BigQuery (Dynamic Table Creation) to be used for various
+ * purposes. e.g. data
+ * analytics, ML model. Cloud DLP inspection and masking can be configured by
+ * the user and can make
+ * use of over 90 built in detectors and masking techniques like tokenization,
+ * secure hashing, date
+ * shifting, partial masking, and more.
+ *
+ * <p>
+ * <b>Pipeline Requirements</b>
+ *
+ * <ul>
+ * <li>DLP Templates exist (e.g. deidentifyTemplate, InspectTemplate)
+ * <li>The BigQuery Dataset exists
+ * </ul>
+ *
+ * <p>
+ * <b>Example Usage</b>
+ *
+ * <pre>
```

Comment:
It is in the README only the creation of the template, it is not explained how to run the job. But this explanation of how to run the job is covered, in a more generic way, not as specific as in this use case, in the PIPELINE_DEPLOYMENTS file.
 In this case, do you think that this information is redundant and it would be better to just remove this example usage from the Java file, without any other changes in other files?